ui/store/actions.js

```diff
@@ -3605,13 +3617,18 @@ export function trackMetaMetricsEvent(payload, options) {
 }
 
 export function createEventFragment(options) {
-  return submitRequestToBackground('createEventFragment', [options]);
+  const actionId = Date.now() + Math.random();
```

Comment:
Doesn't have to be this PR but probably worth adding a method to generate these action ids so that if we change how we do it in the future it can be changed in one place.

Reply:
I updated the PR.